app/serializers/book_serializer.rb

```diff
@@ -1,3 +1,4 @@
 class BookSerializer < ActiveModel::Serializer
-  attributes :id, :genre, :author, :image, :title, :publisher, :year, :actual_rent
+  attributes :id, :author, :title, :image, :publisher, :year,
```

Comment:
check if you don't need a custom serializer for the index endpoint (some times you want to show some info on #index and another one in #show)

check this also for user and rent